foundry.toml

```diff
@@ -26,6 +26,7 @@ src = 'src/interfaces/'
 
 [fuzz]
 runs = 1000
+max_test_rejects = 10000000
 
```

Comment:
10m ? this may hurt on testing times, not important for now

Reply:
If not present it fails with `too many inputs`. I can lower the token amount from 5 to 3 if you want.

Reply:
3 is good enough for now, preferable that tests don't last too much than having 5 tokens
we could somehow create later different setups, where locally you run 3 tokens, 256 runs, 100k max rejects, and on CI we can test 5 tokens, 10k runs, 10M max rejects